docs/gameroom_walkthrough/act_1/behind_the_scenes/behind_scene_1.md

```diff
@@ -0,0 +1,132 @@
+## I-1. Behind scene 1: Alice logs into Acme's Gameroom UI
+
+When a user logs in, the user interface (UI) component of the Gameroom client
+application works with the Gameroom REST API to check the user's email address
+and password. Each Gameroom daemon stores the user credentials in a local
+PostgreSQL database; user passwords are hashed so that they remain secret.
+
+### I-1.1. Acme UI sends authorization request to Gameroom REST API
+
+![](./images/auth_login_acme1.svg)
+
+When Alice clicks Log in, the Acme Gameroom UI hashes the password, then sends
+an authorization request to the Acme Gameroom daemon, gameroomd. The request is
+handled by the Gameroom REST API, which is a part of gameroomd.
+
+``` json
+    POST /users/authenticate
+    {
+      "email": "alice@acme.com",
+      "hashedPassword": "8e066d41...d99ada0d"
+    }
+```
+
+The UI does not reveal the user's password to the REST API because the password
+is used to encrypt signing keys.
```

Comment:
The original doc had cross references (links to other sections of the doc).  I strongly recommend adding them now as plain text.  Later, when the full doc is available, we can activate the links.